sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -595,22 +620,34 @@ extension StorageBlobClient: TransferDelegate {
         didUpdateWithState state: TransferState,
         andProgress progress: Float?
     ) {
-        transferDelegate?.transfer(transfer, didUpdateWithState: state, andProgress: progress)
+        guard let delegate = transferDelegate else { return }
```

Comment:
Since these methods just proxy the transferDelegate message coming in from `URLSessionTransferManager`, it might be better to put these changes there instead.

Reply:
Good point. I'll update.

Reply:
I did the update and realize that means I don't actually need to wrap calls in the operation classes since they will funnel through these methods.